cloud_functions/fastly_announce_purge/src/main.py

```diff
@@ -1,14 +1,25 @@
 import json
 import base64
-import logging
 import os
 
+import logging
+from google.cloud.logging import Client
+from google.cloud.logging.handlers import CloudLoggingHandler
+
 import functions_framework
 from cloudevents.http import CloudEvent
 
 from arxiv.integration.fastly.purge import purge_fastly_keys
 
+#cloud function logging setup
+handler = CloudLoggingHandler(Client())
+functions_framework.setup_logging()
 logger = logging.getLogger(__name__)
+log_level_str = os.getenv('LOG_LEVEL', 'INFO')
+log_level = getattr(logging, log_level_str.upper(), logging.INFO)
+logger.setLevel(log_level)
+logger.addHandler(handler)
+
```

Comment:
What happens if you run this locally? Is there a way to disable this while testing?

Reply:
running it locally it tells me the logs in the terminal it also tells me its collecting some logs and sending them, but I dont believe this goes anywhere, I don't think my computer has enough connection data to be able to send them to gcp, and I dont see them appearing in the cloud function